k9mail-library/src/main/java/com/fsck/k9/mail/ServerSettings.java

```diff
@@ -16,10 +16,13 @@
  * @see com.fsck.k9.mail.store.StoreConfig#getTransportUri()
  */
 public class ServerSettings {
+
+    public enum Type { IMAP, SMTP, WebDAV, POP3 }
```

Comment:
Is WebDAV only for incoming mail? 

In which case I suppose TransportType would be always SMTP.